### PR TITLE
12.0 l10n co edi anexo 18 nla

### PR DESCRIPTION
**Pull request by nla-odoo**
- Update partner document type

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_co/models/res_partner.py [hunk 1]
```diff
@@ -3,19 +3,23 @@
 
 from odoo import api, fields, models
 
+
```
Comment (oco-odoo) [inline addons/l10n_co/models/res_partner.py:6]: Useless diff

**Review by oco-odoo — CHANGES_REQUESTED** [review]
Your commit history feels weird: if you want to reorder the selection values, do it first, then add the new values in a distinct commit. They seem mixed. 

Is there a task about this? Why do we do that ? Your commit message needs to be more explicit, and to reference the original task :) 

I let @jco-odoo merge this PR once it's ready, since he already reviewed the enterprise part

**Comment by jco-odoo** [comment]
@nla-odoo Indeed, would be good to have a good squash here.

**Review by william-andre — COMMENTED** [review]
@nla-odoo I can't find a related ticket, is it still needed?
Version 12.0 is not supported anymore.

**Comment by jco-odoo** [comment]
No compaints for this anymore, so better to close.

**Pull request closed without merging**